plugins/module_utils/common.py

```diff
@@ -496,7 +496,7 @@ def load_resource_definitions(self, src):
         if not os.path.exists(path):
             self.fail(msg="Error accessing {0}. Does the file exist?".format(path))
         try:
-            with open(path, "r") as f:
+            with open(path, "r", encoding = "utf-8") as f:
```

Comment:
```suggestion
            with open(path, "r", encoding="utf-8") as f:
```

Reply:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
I created the changelog and also applied @gravesm suggestion to use rb mode for opening the file rather than guessing the UTF-X